pkgs/tools/misc/whatfiles/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, makeWrapper}:
+
+stdenv.mkDerivation {
+  pname = "whatfiles";
+  version = "2020-06-16";
```

Comment:
If this has no release, please use `unstable` flag.

Reply:
What `unstable` flag? You mean `version = "unstable-2020-06-16";`?

Reply:
Please set `pname` to `whatfiles-unstable` and simply use the date as `version`. See also #68518.